.github/workflows/release-30_publish_release_draft.yml

```diff
@@ -98,7 +110,7 @@ jobs:
           draft: true
 
   publish-runtimes:
-    needs: [ build-runtimes, publish-release-draft ]
+    needs: [ build-runtimes, build-frame-omni-bencher, publish-release-draft ]
```

Comment:
@EgorPopelyaev is this the right approach to add the binary to the release artifacts? is there an internal guide/wiki around this somewhere? If this is broadly correct, I will replicate it in the chain-spec-builder PR.

Reply:
Hey Kian, unfortunately there is no guide around it as so far we had only two binaries that we release from release team. As we discussed in the coordination chat. If we go the same way for the new bins (with build on the cleanroom and signatures with the SEC key, that we use for `polkadot` and` polkadot-parachain`), then shuld be done not here but in one of our scripts [like this one](https://github.com/paritytech/cleanroom/blob/master/ansible/roles/polkadot/files/scripts/polkadot-release-strategy). The main point of these process is so that people could verify that those binaries can be trusted. If it is an overengineering for `omni-bencher` and similar bins. Then they could be build and attached theway it is done here

Reply:
I think omni-bencher and chain-spec-builder can be built without signing, so I will use this approach. 

Thank you for the explanation @EgorPopelyaev!

Reply:
Oki, works for me